orchestra/admin.py

```diff
@@ -25,6 +25,7 @@
 from orchestra.models import TaskAssignment
```

Comment:
high-level: i don't quite understand the difference between `todoqa` and `recommendation`. I get why we need `todoqa`, but why do we need `recommendation`. Can we eliminate the second one?

Reply:
is the idea that recommendations show up in subsequent tasks? if so, send a loom video of that experience.

Reply:
sending you the loom videos for the end to end experience.